canonical-web-and-design/vanilla-framework
# Add individual examples to base styles

## Done

Adds individual example pages for base styles.

Fixes #2736 

## QA

- Pull code
- Run `./run serve --watch` or demo
- Open [Examples](https://vanilla-framework-canonical-web-and-design-pr-2760.run.demo.haus/examples)
- Check if base style examples are available in 'individual CSS' section on the bottom of the page
- Check if 'full Vanilla' base examples work as before
- Check if individual base examples work and look the same as full vanilla examples
